docs/contents/getting-started.md

```diff
@@ -94,6 +94,21 @@ That one is specified on a per project basis with `homepage_contents`, which def
 The homepage typically contains the link to the generated PDF.
 Note that the homepage is only added to the html output and not to pdf or other outputs.
 
+### Creating a Website Landing Page for Your Book
+
+If you want to create a landing page for your book when you host it, you will need to create a `index.md` file in the `contents` directory.
+Then, using an `h1` header from Markdown, give the file a title.
+Immediately after the title, you will need to write `{-}` to indicate that this is the landing page for your book's website.
```

Comment:
`{-}` is a special syntax in pandoc:

https://pandoc.org/MANUAL.html#heading-identifiers

```
Headings with the class unnumbered will not be numbered, even if --number-sections is specified. A single hyphen (-) in an attribute context is equivalent to .unnumbered, and preferable in non-English documents. So,
```